src/containerapp/HISTORY.rst

```diff
@@ -3,6 +3,16 @@
 Release History
 ===============
 
+0.3.6
+++++++
+* Added parameter --environment to 'az containerapp list'
+* Added 'az containerapp revision label swap' to swap traffic labels
+* BREAKING CHANGE: 'az containerapp revision list' now shows only active revisions by default, added flag --all to show all revisions
+* Fixed bug with 'az containerapp up' where custom domains would be removed when updating existing containerapp
+* Fixed bug with 'az containerapp auth update' when using --unauthenticated-client-action
+* BREAKING CHANGE: 'az containerapp env certificate upload' now overwrites the existing certificate by default. Added flag --overwrite to show the prompt for confirmation of overwriting the existing certificate.
```

Comment:
@lil131 update this to say az containerapp env certificate  upload does not prompt by default when re-uploading an existing certificate. Added --show-prompt to show prompts on re-upload.

Update the parameter to --show-prompt instead of overwrite. Thanks!